components/Cards/index.js

```diff
@@ -1,6 +1,28 @@
 // STEP 3: Create Article cards.
 // -----------------------
 // Send an HTTP GET request to the following address: https://lambda-times-backend.herokuapp.com/articles
+const cardsContainer = document.querySelector(".cards-container");
+
+axios
+  .get("https://lambda-times-backend.herokuapp.com/articles")
+  .then(res => {
+    console.log("data: ", res);
+    const articlesObj = res.data.articles;
+    console.log(articlesObj);
+    Object.values(articlesObj).forEach(arr => {
+      const articlesArr = arr;
+      console.log(articlesArr);
+      articlesArr.forEach(articleObj => {
+        console.log(articleObj);
+        const articleComp = createCard(articleObj);
+        cardsContainer.appendChild(articleComp);
```

Comment:
Christian nice approach to getting each of the return axios object values to get the articles for each "category".
Another way to do this would be using a `for..in` statement like so 
loop https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

```
axios
  .get('https://lambda-times-backend.herokuapp.com/articles')
  .then(response => {
    // articles variable to host the articles from response
    let articles = response.data.articles;
    // going through the keys of the articles object 
    for (topic in articles) {
       //  console.log(topic) you should get the articles "categories" === keys
      // . console.log(articles[topics]) you should get the array that that key(topic) is equal too and using that array we can iterate through it using a for each and appending our component from there
      articles[topic].forEach(article => {
        entry.appendChild(createCards(article));
      });
    }
  });
```